### DISCUSSION TITLE
Docker images for .NET 8 are still in Preview - is it possible to ignore them?

### DISCUSSION THREAD

**Discussion by nZeus** [renovatebot/renovate#24395] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello,

Yesterday Microsoft released RC1 version of .NET 8, including docker images for it
<img width="1105" alt="image" src="https://github.com/renovatebot/renovate/assets/2727326/211c481e-98bb-4c8c-8918-145cb1319048">

They also tagged those images as `8.0` which I believe is confusing.
Renovate bot detected the new versions and created PR to upgrade our docker files from `mcr.microsoft.com/dotnet/aspnet:7.0` to `mcr.microsoft.com/dotnet/aspnet:8.0`

I wonder, if it's possible (on renovate side) to treat this `8.0` as a preview version, as it's not the final release yet, so that we don't get this PR until Microsoft officially releases 8.0?

Thanks

### What you tried so far.

-

**Comment by rarkins** [comment, marked as answer]
Yes, you can configure `packageRules` with `allowedVersions`. You could also contribute this as a `workarounds` preset in Renovate itself if you think all users would benefit from it.

**Reply by nZeus** [reply]
Ok, thank you 👍 It's a minor issue, but still I think it makes sense for most projects to wait until the final release

**Reply by rarkins** [reply]
Yes it's a petty they chose to tag an unstable release this way

**Reply by nZeus** [reply]
Well, I think it would be fair to also mention this thread, https://github.com/dotnet/dotnet-docker/issues/4117#issuecomment-1268876809

> Given that .NET RCs have a "go live" license, meaning they are supported for production use, our assertion is that it is appropriate to make them available with the 8.0 tag

**Reply by rarkins** [reply]
Yes, but I think they're unfortunately wrong. Never mind, nothing we can do
